e2e/nomostest/client.go

```diff
@@ -110,16 +112,60 @@ type Cluster interface {
 	Connect() error
 }
 
+func upsertCluster(t testing.NTB, cluster Cluster, opts *ntopts.New) {
+	var adoptCluster bool
+	switch *e2e.CreateClusters {
+	case e2e.CreateClustersEnabled:
+		adoptCluster = false
+	case e2e.CreateClustersLazy:
+		adoptCluster = true
+	default:
+		t.Fatalf("unrecognized value for create-cluster: %s", *e2e.CreateClusters)
```

Comment:
can we do flag validation earlier?

Reply:
Yeah, should be doable. I'll move this earlier

Reply:
done